devicetypes/smartthings/inovelli-dimmer.src/inovelli-dimmer.groovy

```diff
@@ -0,0 +1,606 @@
+		],
+		[
+			name           : "Association Behavior", key: "associationBehavior", type: "enum",
+			parameterNumber: 12, size: 1, defaultValue: 15,
+			values         : [
+				1 : "Local",
+				2 : "3-Way",
+				3 : "3-Way + Local",
+				4 : "Z-Wave HUB",
+				5 : "Z-Wave HUB + Local",
+				6 : "Z-Wave HUB + 3-Way",
+				7 : "Z-Wave HUB + Local",
+				8 : "Timer",
+				9 : "Timer + Local",
+				10: "Timer + 3-Way",
+				11: "Timer + Local",
+				12: "Timer + Z-Wave HUB",
+				13: "Timer + Local",
+				14: "Timer + Z-Wave HUB",
+				15: "All"
+			],
+			description    : "When should the switch send commands to associated devices."
+		],
+		[
+			name           : "LED Indicator Color", key: "ledIndicatorColor", type: "range",
+			parameterNumber: 13, size: 2, defaultValue: 170,
+			range          : "0..255",
+			description    : "This will set the default color of the LED Bar. This is calculated by using a hue color circle (Value / 255 * 360). See website for more info. This parameter can be set without a HUB from the Configuration Button."
+		],
+		[
+			name           : "LED Indicator Intensity", key: "ledIndicatorIntensity", type: "range",
+			parameterNumber: 14, size: 1, defaultValue: 5,
+			range          : "0..10",
+			description    : "This will set the intensity of the LED bar (ie: how bright it is). Example of how the values work: 0 = Off, 1 = Low, 5 = Medium, 10 = High. This parameter can be set without a HUB from the Configuration Button."
+		],
+		[
+			name           : "LED Indicator Intensity (When Off)", key: "ledIndicatorIntensity(WhenOff)", type: "range",
+			parameterNumber: 15, size: 1, defaultValue: 1,
+			range          : "0..10",
+			description    : "This is the intensity of the LED bar when the switch is off. Example of how the values work: 0 = Off, 1 = Low, 5 = Medium, 10 = High. This parameter can be set without a HUB from the Configuration Button."
+		],
+		[
+			name           : "LED Indicator Timeout", key: "ledIndicatorTimeout", type: "range",
+			parameterNumber: 17, size: 1, defaultValue: 3,
+			range          : "0..10",
+			description    : "Changes the amount of time the RGB Bar shows the Dim level if the LED Bar has been disabled. Example of how the values work: 0 = Always off, 1 = 1 second after level is adjusted, 10 = 10 seconds after level is adjusted."
+		],
+		[
+			name           : "Active Power Alerts", key: "activePowerAlerts", type: "boolRange",
+			parameterNumber: 18, size: 1, defaultValue: 10,
+			range          : "1..100", disableValue: 0,
+			description    : "The power level change that will result in a new power report being sent (% of previous report). Example of how the values work: 0 = Disabled, 10 = 10% of previous report, 100 = 100% of previous report."
+		],
+		[
+			name           : "Periodic Power + Energy Reports", key: "periodicPower+EnergyReports", type: "range",
```

Comment:
we should not expose this because it could result in some nasty side-effects (clogging the Z-wave queue, for example)

Reply:
Done